api/src/indigo_object.cpp

```diff
@@ -122,6 +122,10 @@ IndigoObjectTypes::IndigoObjectTypes()
     emplace(IndigoObject::TGROUP, "TGroup");
     emplace(IndigoObject::TGROUPS_ITER, "TGroupsIterator");
     emplace(IndigoObject::GROSS_REACTION, "GrossReaction");
+    emplace(IndigoObject::JSON_LOADER, "JsonLoader");
+    emplace(IndigoObject::JSON_SAVER, "JsonSaver");

```

Comment:
Do we need `JSON_LOADER` and `JSON_SAVER` if the corresponding classes are not used?

Reply:
fixed